src/OperationalInsights/OperationalInsights.Autorest/examples/Get-AzOperationalInsightsDeletedWorkspace.md

```diff
@@ -0,0 +1,18 @@
+### Example 1: {{ Add title here }}
+```powershell
```

Comment:
Please fill the examples and regenerate the doc.

Reply:
@wyunchi-ms  updated docs, do we need to exclude doc folder? 
it contains the same files...